src/features/wallet/chains/sepolia.ts

```diff
@@ -0,0 +1,19 @@
+import { Network } from '../models/network/types/Network';
+
+export const SepoliaChain: Network = {
+  chainId: 11155111,
+  chainName: 'Sepolia',
+  nativeCurrency: {
+    name: 'Sepolia ETH',
+    symbol: 'ETH',
+    decimals: 18,
+  },
+  rpcUrls: ['https://ethereum-sepolia-rpc.publicnode.com'],
+  blockExplorerUrls: ['https://sepolia.etherscan.io/'],
```

Comment:
Could you remove the slash at the end. It needs to be https://sepolia.etherscan.io
Otherwise, the link in the ProfileDropdownMenu.tsx will be broken for this network.